litellm/integrations/langfuse.py

```diff
@@ -213,12 +212,10 @@ def _log_langfuse_v2(
             "output": output,
             "user_id": metadata.get("trace_user_id", user_id),
             "id": metadata.get("trace_id", None),
+            "metadata":  metadata,
         }
         cost = kwargs["response_cost"]
-        print_verbose(f"trace: {cost}")
         if supports_tags:
-            for key, value in metadata.items():
-                tags.append(f"{key}:{value}")
```

Comment:
users asked to see their proxy `api_keys` in the tags. Instead of deleting this can you edit it to include `user_api_key`, `user_api_key_user_id` if they exist in metadata